general/solomiia_tymoshchuk/Task10.java

```diff
@@ -0,0 +1,39 @@
+package general.solomiia_tymoshchuk;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/**
+ * Write a program that simulates the guess a number game.
+ * The program should generate a random number between 1 and N (N should be read from the input)
+ * and the user needs to guess it. If the user enters a number less than guessed, the program needs to output “Larger”.
+ * If the user enters a number greater than guessed - the program should output “Smaller”.
+ * If the user guesses the number correctly - the program should output “That’s it!” and finish.
+ **/
+public class Task10 {
+
+    public String printSentence(String sentence) {
+        System.out.println(sentence);
+        return sentence;
+    }
+
+    public String guessNumber() {
```

Comment:
The user needs to keep guessing till it gets the number. Sorry for the confusion if that wasn't clear from the task description.